react-search-component/my-app/src/RenderedList.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+export default function List({ items }: Props) {
+  const [inputValue, setInputValue] = useState('');
```

Comment:
`inputValue` is not a very descriptive name. It describes where you got the value, not what it is used for. "filter" or "search" would be better terms than "input".